app/controllers/api/v1x0/mixins/index_mixin.rb

```diff
@@ -17,7 +17,8 @@ def collection(base_query, pre_authorized: false)
             :base_query => filtered(scoped(base_query, pre_authorized)),
             :request    => request,
             :limit      => pagination_limit,
-            :offset     => pagination_offset
+            :offset     => pagination_offset,
+            :sort_by    => query_sort_by
```

Comment:
Should this be moved to the v1.1 version of the mixin?

Reply:
Yes, once it's merged. Whichever one goes first should update it.

Reply:
Is this `query_sort_by` some rails magic for `params.permit(:sort_by)`? I can't seem to find documentation for it and it seems weird to bypass our standard of `.permit` or `.require` (even though this one isn't required, of course).

Reply:
It's in common actually, so we have it here: 
https://github.com/RedHatInsights/insights-api-common-rails/blob/f0b86e82f870bb36b798a1dbccbb73edf52f8b5f/lib/insights/api/common/application_controller_mixins/parameters.rb#L112-L114

Reply:
I see what @syncrou was saying now - I need to make a new index_mixin for 1.1, I didn't notice the versioning stuff was merged.

Reply:
@lindgrenj6 Gotcha 👍

Reply:
Do we still need this in `v1x0`?